package.json

```diff
@@ -17,7 +17,7 @@
     "express-graphql": "0.3.0",
     "graphql": "0.4.2",
     "graphql-relay": "0.3.2",
-    "react": "^0.14.0-rc",
+    "react": "^0.14.0",
     "react-dom": "^0.14.0-rc",
```

Comment:
Actually, would you mind backing out this part of the change, as it's not related to the key warning. I just merged https://github.com/relayjs/relay-starter-kit/pull/42 for the dependency updates instead, as it handles `react`, `react-dom` and `react-relay`.